### PR TITLE
Add ignored test that simulates 16k vault creations + transfers 

### PR DESCRIPTION
**Pull request by KentShikama**
## Overview
<sup>_What this PR does, and why it's needed_</sup>

As suggested by @ArturGajowy during standup.

### JIRA ticket:
<sup>_Create it if there isn't one already._</sup>

### Notes
<sup>_Optional. Add any notes on caveats, approaches you tried that didn't work, or anything else._</sup>

### Please make sure that this PR:
- [x] is at most 200 lines of code (excluding tests),
- [x] meets [RChain development coding standards](https://rchain.atlassian.net/wiki/spaces/DOC/pages/28082177/Coding+Standards),
- [x] includes tests for all added features,
- [x] has a reviewer assigned,
- [x] has [all commits signed](https://rchain.atlassian.net/wiki/spaces/DOC/pages/498630673/How+to+sign+commits+to+rchain+rchain).

### [Bors](https://bors.tech/) cheat-sheet:

- `bors r+` runs integration tests and merges the PR (if it's approved),
- `bors try` runs integration tests for the PR,
- `bors delegate+` enables non-maintainer PR authors to run the above.

### CODE REVIEW

File: casper/src/test/scala/coop/rchain/casper/RholangBuildTest.scala [hunk 1]
```diff
@@ -60,4 +74,40 @@ class RholangBuildTest extends FlatSpec with Matchers {
...
+                Vault(RevAddress.fromEthAddress(ethAddress), initRevBalanceStr.toLong)
+            }
+          }
+          .toSeq
+        val genesisPk = defaultPub
+
+        import node._
+
+        for {
+          deploy <- ConstructDeploy.sourceDeployNowF(
```
Comment (JosephDenman) [inline casper/src/test/scala/coop/rchain/casper/RholangBuildTest.scala]: Here you're re-deploying `RevGenerator`. If the performance of this deploy, with and without the changes from the parallelization PR I opened, is what you're measuring, then that explains the performance disparity. I'm measuring the time to compute the genesis block with 16K vaults. You're measuring the time to compute `RevGenerator` with 16K vaults.

File: casper/src/test/scala/coop/rchain/casper/RholangBuildTest.scala [hunk 2]
```diff
@@ -1,19 +1,33 @@
...
+import coop.rchain.shared.Log
 import monix.execution.Scheduler.Implicits.global
 import org.scalatest.{FlatSpec, Matchers}
 
+import scala.io.Source
+import scala.util.{Failure, Success, Try}
+
 class RholangBuildTest extends FlatSpec with Matchers {
 
   val genesis = buildGenesis()
```
Comment (JosephDenman) [inline casper/src/test/scala/coop/rchain/casper/RholangBuildTest.scala:35]: Here you're constructing the entire genesis block.

File: casper/src/test/scala/coop/rchain/casper/RholangBuildTest.scala (hunk 1 above)
Comment (KentShikama) [reply casper/src/test/scala/coop/rchain/casper/RholangBuildTest.scala]: It shouldn't make a difference.

File: casper/src/test/scala/coop/rchain/casper/RholangBuildTest.scala (hunk 2 above)
Comment (KentShikama) [reply casper/src/test/scala/coop/rchain/casper/RholangBuildTest.scala:35]: It is created with I think 4 deploys and takes only a few seconds.

Comment (ArturGajowy) [reply casper/src/test/scala/coop/rchain/casper/RholangBuildTest.scala:35]: yeah - the wallets should be passed in here as parameter to genesisParameters, that are passed to buildGenesis

Comment (ArturGajowy) [reply casper/src/test/scala/coop/rchain/casper/RholangBuildTest.scala:35]: (see the default parameters - we need to pass them explicitly)

**Comment by ArturGajowy** [comment]
Instead of including the wallets.txt, please generate the data on the fly. If you use any randomness for that - use a fixed RNG seed

**Comment by KentShikama** [comment]
It unfortunately puts a dependency on the Ethereum blockchain so I don't think that is reasonable

**Comment by JosephDenman** [comment]
> It unfortunately puts a dependency on the Ethereum blockchain so I don't think that is reasonable

They can be randomly generated addresses.

**Comment by KentShikama** [comment]
> They can be randomly generated addresses.

Sounds good

**Comment by KentShikama** [comment]
PTAL @ArturGajowy @JosDenmark

File: casper/src/test/scala/coop/rchain/casper/RholangBuildTest.scala [hunk 3]
```diff
@@ -60,4 +76,40 @@ class RholangBuildTest extends FlatSpec with Matchers {
...
+          minimumBond = 0L,
+          maximumBond = Long.MaxValue,
+          epochLength = 1,
+          validators = bonds.map(Validator.tupled).toSeq
+        ),
+        genesisPk = defaultPub,
+        vaults = vaults,
+        supply = Long.MaxValue
+      )
+    )
```
Comment (ArturGajowy) [inline casper/src/test/scala/coop/rchain/casper/RholangBuildTest.scala]: there's no need to duplicate all the default values. You can simply:
```
val genesisParams = defaultGenesisParams().copy(vaults = vaults)
```

Comment (KentShikama) [reply casper/src/test/scala/coop/rchain/casper/RholangBuildTest.scala]: PTAL

**Comment by KentShikama** [comment]
bors try

**Review by ArturGajowy — APPROVED** [review]

**Review by JosephDenman — APPROVED** [review]

**Pull request merged**